docker/ubuntu-based/dev/build-vpp.sh

```diff
@@ -3,9 +3,14 @@
 set -euo pipefail
 
 cd /opt/vpp-agent/dev/
-git clone https://gerrit.fd.io/r/vpp
+
+#git clone https://gerrit.fd.io/r/vpp
```

Comment:
Perhaps these commented lines can go away?

Reply:
later on, we may go back to this location, but yes, is is not needed